x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -99,6 +99,8 @@ describe('Task Runner', () => {
     internalSavedObjectsRepository: savedObjectsRepositoryMock.create(),
     alertTypeRegistry,
     kibanaBaseUrl: 'https://localhost:5601',
+    supportsEphemeralTasks: true,
```

Comment:
should we add tests for when `supportsEphemeralTasks` is `false` as well?

Reply:
Sounds good!